### PR TITLE
Support internlm2

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: docs/en/advance/long_context.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Context length extrapolation
```
Comment (RunningLeon) [inline docs/en/advance/long_context.md:1]: may add to `index.rst` to update this doc in readthedocs

File: docs/zh_cn/advance/long_context.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# 长文本外推
```
Comment (RunningLeon) [inline docs/zh_cn/advance/long_context.md:1]: may add to `index.rst` to update this doc in readthedocs

File: docs/en/advance/long_context.md (hunk 1 above)
Comment (lvhan028) [reply docs/en/advance/long_context.md:1]: OK, added.

**Review by RunningLeon — COMMENTED** [review]
LGTM

**Review by RunningLeon — APPROVED** [review]

**Review by AllentDan — APPROVED** [review]

**Pull request merged**